spider_feeder/parser.py

```diff
@@ -0,0 +1,38 @@
+import json
+from csv import DictReader
+
+from scrapy.exceptions import NotConfigured
+
+
+class TxtParser:
+
+    def __init__(self, settings):
```

Comment:
Not if would be better to declare the factory methods `from_settings` or `from_crawler` and then use `create_instance` from scrapy utils.